modules/test/helpers/common.go

```diff
@@ -42,13 +42,24 @@ func NewTestHelper(
 	return &TestHelper{
 		ctx:       ctx,
 		k8sClient: k8sClient,
-		timeout:   timeout,
+		timeout:   getTestTimeout(timeout),
 		interval:  interval,
 	}
 }
 
+// getTestTimeout returns test timeout from TEST_TIMEOUT_SEC environment
+// variable, in seconds; or picks defaultTimeout, in milliseconds
+func getTestTimeout(defaultTimeout time.Duration) time.Duration {
+	t := os.Getenv("TEST_TIMEOUT_SEC")
+	timeout, err := strconv.Atoi(t)
+	if err != nil || t == "" {
```

Comment:
if t == "" then Atio(t) has already been an error. So here you don't need to check for t any more.
https://go.dev/play/p/t4zMj83BoKe?v=goprev